docs/quickstart.rst

```diff
@@ -264,12 +264,12 @@ The following two rules differ in their use of a trailing slash. ::
         return 'The about page'
 
 The canonical URL for the ``projects`` endpoint has a trailing slash.
-It's similar to a folder in a file system. If you access the URL without
+It's similar to a folder in a file system. If you access ``/projects`` - the URL without
 a trailing slash, Flask redirects you to the canonical URL with the
-trailing slash.
+trailing slash - ``/projects/``.
```

Comment:
It will be better to make the other tips' position consistent with this one. For example, use `Accessing the URL with a trailing slash (``/about/``)` instead of `Accessing the URL (``/about/``) with a trailing slash`.